docs/run-a-validator.md

```diff
@@ -0,0 +1,340 @@
+- <strong>Memory</strong> - 64GB
+
+The specs posted above are by no means the minimum specs that you could use when running a validator, however you should be aware that if you are using less you may need to toggle some extra optimizations in order to be equal to other validators that are running the standard.
+
+### Node Prerequisites: Install Rust and Dependencies
+
+Once you choose your cloud service provider and set-up your new server, the first thing you will do is install Rust.
+
+If you have never installed Rust, you should do this first.
+
+If you have already installed Rust, run the following command to make sure you are using the latest version.
+
+```
+rustup update
+```
+
+If not, this command will fetch the latest version of Rust and install it.
+
+```
+curl https://sh.rustup.rs -sSf | sh -s -- -y
+```
+
+> If you do not have "curl" installed, run "sudo apt install curl"
+
+To configure your shell, run the following command.
+
+```
+source $HOME/.cargo/env
+```
+
+Verify your installation.
+
+```
+rustc --version
+```
+
+Finally, run this command to install the necessary dependencies for compiling and running the Polkadex node.
+
+```
+sudo apt install make clang pkg-config libssl-dev build-essential
+```
+
+Note - if you are using OSX and you have Homebrew installed, you can issue the following equivalent command INSTEAD of the previous one:
+
+```
+brew install cmake pkg-config openssl git llvm
+```
+
+### Install & Configure Network Time Protocol (NTP) Client
+
+[NTP](https://en.wikipedia.org/wiki/Network_Time_Protocol) is a networking protocol designed to synchronize the clocks of computers over a network. NTP allows you to synchronize the clocks of all the systems within the network. Currently it is required that validators' local clocks stay reasonably in sync, so you should be running NTP or a similar service. You can check whether you have the NTP client by running:
+
+If you are using Ubuntu 18.04 / 20.04, NTP Client should be installed by default.
+
+```
+timedatectl
+```
+
+If NTP is installed and running, you should see `System clock synchronized: yes` (or a similar message). If you do not see it, you can install it by executing:
+
+```
+sudo apt-get install ntp
+```
+
+ntpd will be started automatically after install. You can query ntpd for status information to verify that everything is working:
+
+```
+sudo ntpq -p
+```
+
+### Building and Installing the `Polkadex` binary
+
+You will need to build the `Polkadex` binary from the [Polkadex-Substrate/Polkadex](https://github.com/Polkadex-Substrate/Polkadex) repository on GitHub using the source code available in the v0.4.0 release.
+
+```
+git clone https://github.com/Polkadex-Substrate/Polkadex.git
+cd Polkadex
+```
+
+Build native code with the cargo release profile.
+
+```
+git checkout v0.4.0
+./scripts/init.sh
```

Comment:
Must be updated only fetches wasm for nightly not nightly 20210511 which we are pinned to